projects/compiler/test/compiler/arrays.abra

```diff
@@ -295,14 +295,13 @@ func exclaim(i: Int, _: Int, x = "!"): String = i + x //"$i$x"
   println(arr.map((i, _, x = "!") => i + x))
 })()
 
-// TODO: there's something up with printing an array of this particular size
-// // Array#flatMap
-// (() => {
-//   val arr = [1, 2, 3, 4]
+// Array#flatMap
```

Comment:
Pretty nice that this test passes, now that I finally addressed that nefarious bug